contents/tutorials/churn-rate.md

```diff
@@ -0,0 +1,87 @@
+---
+title: How to calculate and lower churn rate with PostHog
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-09-28
+featuredImage: ../images/tutorials/banners/actions.png
```

Comment:
I've sent you a Figma invite. Shout when you're free and I'll give a quick tutorial on how to create new tutorial images.